_posts/2013-05-11-skip-nav-links.md

```diff
@@ -3,39 +3,52 @@ layout: post
 ---
-**Short answer:** Use skip nav links in conjunction with a coherent heading outline and [ARIA landmarks](https://a11yproject.com/posts/aria-landmark-roles/). You may need to also implement a JavaScript polyfill for Webkit based browsers.
+**Short answer:** Use skip navigation links in conjunction with a coherent heading outline and [ARIA landmarks](/posts/aria-landmark-roles/).
 
 ---
 
-It can be frustrating and fatiguing for folks with limited mobility to have to have to repeatedly tab through navigation links to get to the main content of a page. People who use screen readers face similar frustration when the page outline is not well defined. In order to address this issue, WCAG 2.0 has specified a [guideline for bypassing repetitive blocks of content](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html). One technique [recommended by the W3C](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G1) is to include a skip navigation link at the beginning of the page, that changes focus to the first element after the repeated content.
+People who navigate by keyboard have to tab through many links to get past repeated content.  People using screen readers are stuck listening to navigation links on every page of your site. This causes frustration and fatigue. Use skip navigation links to upgrade user experience for keyboard and screen reader navigators.
 
-Skip nav links are useful for users who use keyboard navigation only, but screen readers now support more sophisticated ways of navigating regions. Specifically, they support [heading navigation](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/H69) and [ARIA landmarks](https://www.w3.org/WAI/intro/aria). You should take advantage of these features by using a clear heading outline and defining page regions, as illustrated in [Quick Tip: ARIA Landmark Roles](https://a11yproject.com/posts/aria-landmark-roles/).
+## When to use
+Skip navigation links are useful on pages with a lot of links and repeated content. WCAG 2.0 specified a [guideline for bypassing repetitive blocks of content](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html). One technique [recommended by the W3C](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G1) is to include a skip navigation link at the beginning of the page. 
+
+## How to use
+Skip navigation links should: 
+- be visible only when focused by a keyboard or screen reader
+- be the first focusable element on the page
+- communicate that it links to the main content or skips navigation
+- bring focus to the main content of the page when activated
+
+Place an `a` element with an `href` pointing to the main content in the body of your HTML.
 
 ### Example
+The A11Y Project has prepared more robust [CSS and HTML code for your own skip navigation link](https://codepen.io/joe-watkins/pen/rjhiK), available on codepen.
 
-{% highlight html %}
+```html
 <body>
   <a href="#main">Skip to main content</a>
```

Comment:
ideally this would be within the `header` so as to not cause automated testing warnings of "all content must be within a landmark".

how important that really is can be up for debate, but would rather err on the side of an example that won't cause a warning.